packages/frontend/src/components/common/GlobalAlert.js

```diff
@@ -224,11 +224,10 @@ const GlobalAlertNew = ({ globalAlert, actionStatus, clearGlobalAlert, closeIcon
                                                     {noTranslationFound
                                                         ? <Translate id={`reduxActions.default.${alert.success ? 'success' : 'error'}`} />
                                                         : <SafeTranslate id={alert.messageCode} data={alert.data} />
```

Comment:
In debugging why the `publicKey` could not be interpolated into the translation string, I noticed that `SaveTranslate` appears to be receiving the wrong argument here. When I replaced `alert.data` with `alert.data.data`, the interpolation worked.

I assume this is affecting other alerts that are potentially failing to display interpolated values in translation text, but given that:
- it wasn't the ideal approach for this issue
- it's out of scope for this PR
- it could potentially break other errors across the wallet
I decided against making that change as part of this fix. However if it is a known issue affecting other parts of the app then we should address it separately.

Reply:
Can you please create a ticket so that we don't forget to follow up on this?

cc @marcinbodnar

Reply:
Done #2504